types.ts

```diff
@@ -42,20 +45,20 @@ export interface Broadcaster {
 };
 
 export interface PriceDataFetched {
-  symbol: string;
+  symbol: string; //symbol or token?
   value: number;
 };
 
 export interface PriceDataBeforeAggregation {
   id: string;
-  symbol: string;
+  symbol: string; //token or symbol or symbolToken? :-)
   source: { [sourceName: string]: number };
   timestamp: number;
   version: string;
 };
 
 export interface PriceDataAfterAggregation extends PriceDataBeforeAggregation {
-  value: number;
+  value: number; //TODO: rename to "aggregatedValue"?
```

Comment:
I am not sure if this change will help a lot. It would also cause troubles with historical redstone data from the last 1.5 months. @jakub-wojciechowski what do you think?